x-pack/plugins/apm/server/routes/services/get_service_instances/get_service_instances_transaction_statistics.ts

```diff
@@ -136,7 +174,10 @@ export async function getServiceInstancesTransactionStatistics<
         field: SERVICE_NODE_NAME,
         missing: SERVICE_NODE_NAME_MISSING,
         ...(size ? { size } : {}),
-        ...(isComparisonSearch ? { include: serviceNodeIds } : {}),
+        ...(serviceNodeIds?.length ? { include: serviceNodeIds } : {}),
+        ...(sortField && sortField !== 'errorRate'
+          ? { order: getOrderInstructions(sortField, sortDirection) }
+          : {}),
       },
       aggs: isComparisonSearch
```

Comment:
I think we call it "detailed statistics" instead of "comparison" now to avoid confusing it with the comparison feature.

```suggestion
      aggs: isDetailsStatisticsSearch
```

Reply:
I think I'll call it `withTimeseries`, WDYT?